src/pages/EditMonitor.vue

```diff
@@ -702,7 +702,7 @@
                             </template>
 
                             <!-- HTTP Options -->
-                            <template v-if="monitor.type === 'http' || monitor.type === 'keyword' || monitor.type === 'json-query' ">
+                            <template v-if="monitor.type === 'http' || monitor.type === 'keyword' || monitor.type === 'json-query' || monitor.type === 'push' ">
```

Comment:
Why did you leave this change? Am I missing something?


```suggestion
                            <template v-if="monitor.type === 'http' || monitor.type === 'keyword' || monitor.type === 'json-query' ">
```

Reply:
My misstake, this is not needed after the method check was remove.